### ISSUE TITLE
Add_text onto 2 lines

### ISSUE THREAD

**Issue by fherbet** [Maker-Melissa/OpenSign#1]
Hi Melissa, 

i've been playing with your lib for a couple of days and and it's really great.. i've been able to easily run really nice animations..
thanks for this !

there's one thing i haven't figured out, it's displaying static text onto 2  lines...
2nd one is somehow cut ...
i tried to use x_offset + y_offset like in below code.. but not sure this is the right way

```
       heure = time.strftime("%H:%M:%S")
        # print(heure)
        message1 = OpenSignCanvas()
        message1.add_font("dejavu", "/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf", 14)
        message1.set_stroke(1, (0, 0, 0))
        message1.add_text(heure, color=(255, 0, 0))
        message1.add_text("temp", color=(0, 0, 255), x_offset=-64, y_offset=-6)

        sign.set_background_image("/home/pi/images/background-matrix-style3.jpg")    #
        sign.set_position(message1, x=-1, y=10)
        sign.show(message1)
        time.sleep(1)
```
can you point me into the right direction if this is something achievable?

thanks very much!
Greetings from FR!

\\ Francois

**Comment by makermelissa** [comment]
Oh yeah, use the cursor property for now. You may need to use the clear() function as well to clear the content on the canvas and then re-add it if you are updating things like the time. One of the things I want to add soon is to detect a newline in the text and automatically move the cursor to the start of the second line.

**Comment by fherbet** [comment]
> Oh yeah, use the cursor property for now. You may need to use the clear() function as well to clear the content on the canvas and then re-add it if you are updating things like the time. One of the things I want to add soon is to detect a newline in the text and automatically move the cursor to the start of the second line.

thanks for your prompt reply... going to try this! 
i've got my clock working but want to add also temperatures from my domotic server

i might ask you some more dumb questions in the future ! sorry about this ;-)

**Comment by fherbet** [comment]
Sorry for being a pain...


would you use 2 message in that case or only 1 ? also how to use cursor property ?

```
        message1 = OpenSignCanvas()
        message1.add_font("dejavu", "/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf", 8)
        message1.set_stroke(1, (0, 0, 0))
        message1.add_text("toto", color=(255, 0, 0))

        message2 = OpenSignCanvas()
        message2.add_text("tata", color=(255, 0, 0))
        message2.cursor()

        sign.set_background_image("/home/pi/images/background-matrix-style3.jpg")    #
        sign.show(message1)
        sign.show(message2)
```

thanks for your help!

\\ Francois

**Comment by makermelissa** [comment]
Oh you're not being a pain. :) You would just use one message, so something like this:
```
        message1 = OpenSignCanvas()
        message1.add_font("dejavu", "/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf", 8)
        message1.set_stroke(1, (0, 0, 0))
        message1.add_text("toto", color=(255, 0, 0))
        message1.cursor = (0, 15)
        message1.add_text("tata", color=(255, 0, 0))

        sign.set_background_image("/home/pi/images/background-matrix-style3.jpg")    #
        sign.show(message1)
```

**Comment by makermelissa** [comment]
Of course the exact location of where you want to move the cursor depends on your sign dimensions, so maybe you want more like (0, 10).

**Comment by makermelissa** [comment]
Fixed in 0.8.4.
